src/components/NewWorkspaceModal.test.ts

```diff
@@ -147,6 +147,33 @@ describe('NewWorkspaceModal', () => {
     screen.getByText('You need a billing project to create a new workspace.');
   });
 
+  it('shows a message if there are no protected billing projects to use for creating a workspace with additional security monitoring ', async () => {
+    // Arrange
+    asMockedFn(Ajax).mockImplementation(
+      () =>
+        ({
+          Billing: {
+            listProjects: async () => [azureBillingProject],
+          },
+          ...nonBillingAjax,
+        } as AjaxContract)
+    );
+
+    // Act
+    await act(async () => {
+      render(
+        h(NewWorkspaceModal, {
+          requireEnhancedBucketLogging: true,
+          onSuccess: () => {},
+          onDismiss: () => {},
+        })
+      );
+    });
+
+    // Assert
+    screen.getByText('You do not have access to a billing project with additional security monitoring.');
```

Comment:
Based this on:

![image](https://github.com/DataBiosphere/terra-ui/assets/484484/8a34f8b1-8896-41d6-b304-476ba8d42495)